src/tests/services/users-service.test.ts

```diff
@@ -0,0 +1,102 @@
+import { expect, test, describe } from '@jest/globals';
+import { faker } from '@faker-js/faker';
+import bcrypt from 'bcrypt';
+
+import { prismaMock } from 'tests/prismaMock';
+import { errors } from 'utils/errors';
+import { ApiError } from 'utils/apiError'; 
+import { UserCreationParams } from 'types/user';
+import db from '../../../prisma/db';
+
+const genPassword = async (password: string) => {
+  const salt = await bcrypt.genSalt(10);
+  return bcrypt.hash(password, salt);
+};
+
+const userInfo = { 
+  id: 1,
+  email: faker.internet.email(),
+  name: faker.person.firstName(),
+  lastname: faker.person.lastName(),
+};
+
+describe('User service: ', () => {
+  
+  describe('Get User', () => {
+    test('success', async () => {
```

Comment:
You can use it instead of test (as you did in the other file).
It is an alias for test.

Reply:
also, pls follow the same structure for the description as you did in the other file :hugs: